src/index.ts

```diff
@@ -61,6 +61,12 @@ const loadFile = (filePath: string): Record<string, unknown> => {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
       const config = require(filePath);
 
+      if (filePath.includes('config.local')) {
```

Comment:
It's possible to have local config files with other names like `config.development.local.js` so we should handle those cases as well

Reply:
Updated to use regex to match files. /config.+local/